CIME/Servers/wget.py

```diff
@@ -26,20 +26,21 @@ def wget_login(cls, address, user="", passwd=""):
             args += "--password {} ".format(passwd)
 
         try:
-            err = run_cmd("wget {} --spider {}".format(args, address), timeout=60)[0]
-        except:
+            err, _, errstr = run_cmd(
+                "wget {} --spider {}".format(args, address), timeout=60
+            )
+        except RuntimeError:
             logger.warning(
-                "Could not connect to repo '{0}'\nThis is most likely either a proxy, or network issue .(location 1)".format(
+                "Could not connect to repo '{0}'\nThis is most likely either a proxy, or network issue .(location 1) ".format(
                     address
                 )
             )
             return None
-
-        if err and not "storage.neonscience.org" in address:
+        if "connected" in errstr:
+            logger.warning("spider mode disabled, server otherwise okay")
+        elif err:
             logger.warning(
-                "Could not connect to repo '{0}'\nThis is most likely either a proxy, or network issue .(location 2)".format(
```

Comment:
Is the bit about "a proxy, or network issue" no longer true? It does seem like a network problem going to be common case, and you just need to retry it . And someone can screw it up with a proxy, so might still be helpful to still suggest it as a possible culprit.

Reply:
Ahh, OK you added that back in again. Cool.